File: doc/src/scripts/tutorials/parts/ControlFromHere.java
import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.SpaceCenter;
import krpc.client.services.SpaceCenter.Part;
import krpc.client.services.SpaceCenter.Vessel;

import java.io.IOException;

public class ControlFromHere {
    public static void main(String[] args) throws IOException, RPCException {
        Connection connection = Connection.newInstance();
        Vessel vessel = SpaceCenter.newInstance(connection).getActiveVessel();
        Part part = vessel.getParts().withTitle("Clamp-O-Tron Docking Port").get(0);
        vessel.getParts().setControlling(part);
        connection.close();
    }
}

File: doc/src/scripts/tutorials/parts/ControlFromHere.py
import krpc

conn = krpc.connect()
vessel = conn.space_center.active_vessel
# with_title matches the localized display name; for language-independent code
# use the internal name instead, e.g. parts.with_name("dockingPort2").
part = vessel.parts.with_title("Clamp-O-Tron Docking Port")[0]
vessel.parts.controlling = part
